packages/ui-react/src/Container/Demo.tsx

```diff
@@ -13,26 +13,30 @@ type DemoProps = {
   component: string
 };
 
-export default function Demo ({ attrs, attrsReq, children, component }: DemoProps) {
-  const allAttrs = Object.assign({ className: '"..."', style: '{...}' }, attrs);
-  const attrsText = Object
-    .keys(allAttrs)
-    .map((attr) => {
-      const value = allAttrs[attr];
-      const bool = value === 'bool';
-      const opt = bool || !attrsReq.includes(attr);
+// ({ attrs, attrsReq, children, component }: DemoProps) {
+export default class Demo extends React.PureComponent<DemoProps> {
+  render () {
+    const { attrs, attrsReq, children, component } = this.props;
 
-      return `${opt ? '[' : ''}${attr}${bool ? '' : '='}${bool ? '' : value}${opt ? ']' : ''}`;
-    })
-    .sort()
-    .join('\n  ');
+    const allAttrs = Object.assign({ className: '"..."', style: '{...}' }, attrs);
+    const attrsText = Object
+      .keys(allAttrs)
+      .map((attr) => {
+        const value = allAttrs[attr];
+        const bool = value === 'bool';
+        const opt = bool || !attrsReq.includes(attr);
 
-  return (
-    <Container className='ui--DemoContainer'>
-      <pre className='ui--DemoHeader'>
-        {`<${component}\n  ${attrsText}\n/>`}
-      </pre>
-      {children}
-    </Container>
-  );
+        return `${opt ? '[' : ''}${attr}${bool ? '' : '='}${bool ? '' : value}${opt ? ']' : ''}`;
+      })
+      .sort()
+      .join('\n  ');
+    return (
```

Comment:
Break before. (The last const declaration is dense - nothing changed from original, but not very maintainable)